app/_src/gateway/plugin-development/quickstart.md

```diff
@@ -0,0 +1,478 @@
+    configurable values to the table and show how to configure the plugin at runtime.
+    
+    At this point, we have a valid plugin (which does nothing). Next We will add logic to the plugin
+    and then see how to validate it.
+
+1. **Add handler logic**
+
+    Plugin logic is defined to be executed at several key points in the lifecycle of
+    HTTP requests, tcp streams, and {{site.base_gateway}} itself.
+
+    Inside the `handler.lua` module, you will add 
+    [functions](/gateway/{{page.release}}/plugin-development/custom-logic/#available-contexts)
+    with well known names to the plugin table, indicating at what points the plugin logic should 
+    be executed. 
+
+    In this example we will add an `response` function, which is executed after a response has been
+    received from an upstream service but before returning it to the client. 
+
+    Let's add a header to the response prior to returning it to the client. Add the following 
+    function implementation to the `handler.lua` file before the `return MyPluginHandler` statement:
+
+    ```lua
+    function MyPluginHandler:response(conf)
+        kong.response.set_header("X-MyPlugin", "response")
+    end
+    ```
+
+    The `kong.response` module provided in the 
+    [Kong PDK](/gateway/{{page.release}}/plugin-development/pdk/), provides
+    functions for manipulating the response sent back to the client. The code above sets 
+    a new header on all responses with the name `X-MyPlugin` and value of `response`. 
+
+    The full `handler.lua` file listing now looks like this:
+
+    ```lua
+    local MyPluginHandler = {
+      PRIORITY = 1000,
+      VERSION = "0.0.1",
+    }
+
+    function MyPluginHandler:response(conf)
+        kong.response.set_header("X-MyPlugin", "response")
+    end
+ 
+    return MyPluginHandler
+    ```
+
+1. **Install Pongo**
+
+    [Pongo](https://github.com/Kong/kong-pongo) is a tool to help you validate and 
+    distribute custom plugins for {{site.base_gateway}}. Pongo uses Docker to
+    bootstrap a {{site.base_gateway}} environment allowing you to quickly load your plugin, 
+    run automated testing, and manually validate the plugin's behavior against
+    various {{site.base_gateway}} versions.
+
+    The following script can automate the installation of Pongo for you, however, if you prefer 
+    you can follow the [manual installation instructions](https://github.com/Kong/kong-pongo?tab=readme-ov-file#installation)
+    instead.
+    
+    If you already have Pongo intalled, you can skip to the next step or run the install script
+    to update Pongo to the latest version.
+   
+    Run the following to install / update Pongo:
+
+    ```sh
+    curl -Ls https://get.konghq.com/pongo | bash
+    ```
+
+    After successfully installing Pongo, ensure that the `pongo` command is 
+    available on your `PATH` by running the command within our project directory:
+
+    ```sh
+    pongo help
```

Comment:
@rspurgeon As I was running through this, I got stuck at this step with a command not found: pongo. Should we tell users how to add it to their path and then tell them to run `pongo help`?

Reply:
@cloudjumpercat The PATH requirement is included in the referenced script output.

```
└─❯❯ curl -Ls https://get.konghq.com/pongo | bash

This script will install the Kong Pongo project to:
  /Users/rick.spurgeon@konghq.com/.kong-pongo

A symbolic link will be created at ~/.local/bin/pongo

If necessary, add the following to your .bashrc or .zshrc file
to put the pongo command on the path:
  export PATH=$PATH:~/.local/bin

Continue? (y/n)
```

and in the manual instructions: https://github.com/Kong/kong-pongo?tab=readme-ov-file#installation

But I will try and further clarify within this text.